### ISSUE TITLE
Improve Windows install instructions

### ISSUE THREAD

**Issue by fingolfin** [oscar-system/oscar-website#150]
For Windows users, our install instructions are lacking: we do tell them to install WSL, but then not how to use that. In particular, when they get to step 2, it is not clear that they need to download the Linux binaries (and then what to do with them!) so naturally they may end up downloading the Windows binaries, which is wrong, but hard to understand.

We should improve this. E.g. by editing Step 2 to explicitly separate between Windows and non-Windows users, and providing more details for the former.

Alternatively, we could have the Windows instructions completely separate from the Unix one. I am open to either, as long as it is clear for Windows users.

Michael also suggested that it might be also helpful to add screenshots to illustrate the process, which indeed seems useful.

**Comment by mohamed-barakat** [comment]
I borrowed, modified, and simplified the Oscar installation instructions for my computer algebra course. Here is the result:

https://homalg-project.github.io/docs/installation

I originally pointed them to the Oscar installation page and the Windows users, which are most of them, ended up downloading Julia for Windows, independent of the WSL (the WSL Link instructed them to install a Ubuntu distribution, so that was OK). After I started this CapAndHomalg installation page the only problem left for some of them was that the virtualization needed by WSL was not activated on their machines. These few people had to activate the virtualization in their BIOS.
